cmd/installer/pkg/install/install.go

```diff
@@ -276,6 +278,38 @@ func (i *Installer) Install(seq runtime.Sequence) (err error) {
 		}
 	}
 
+	if i.options.Save {
```

Comment:
i think this should actually be implicit ... no need for an extra flag

Reply:
if we make this implicit, i don't like that we still attempt to parse the url ... due to parsing ambiguities, parsing non-urls like 'none' works but still doesnt make me feel good about it